components/collaboration/card-components/CollaborationDetails.tsx

```
@@ -67,30 +68,22 @@ const CollaborationDetails: FC<CollaborationDetailsProps> = ({
+              chipText={content}
+              chipIcon={
+                content === "Instagram"
+                  ? faInstagram
+                  : content === "Facebook"
+                  ? faFacebook
+                  : content === "Youtube"
+                  ? faYoutube
+                  : faInstagram
+              }
+            />
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)